include/retdec/fileformat/types/export_table/export_table.h

```diff
@@ -21,14 +21,20 @@ class ExportTable
 {
 	private:
 		using exportsIterator = std::vector<Export>::const_iterator;
-		std::vector<Export> exports; ///< stored exports
+		std::vector<Export> exports;				///< stored exports
```

Comment:
We use tabs only to indent code. If you want to add comments at the end of the line, align it with spaces please.